### DISCUSSION TITLE
按群晖教程弄了几次都是显示"400 Bad Request, The plain HTTP request was sent to HTTPS port"

### DISCUSSION THREAD

**Discussion by chsmin** [natfrp/wiki#247] [使用问题]
网址处输入的是https
本地 IP处试了172.17.0.1和本地群晖ip 192.168.1.5都不行
端口是5001

甚至试用了白银套餐的线路都失败了.

**Comment by fakeboboliu** [comment]
请每一步都完全跟随教程进行，不要动教程中没有碰的任何其他选项，尤其是自动https

**Reply by chsmin** [reply]
感谢, 我就是https那里一开始切到自动后来又切禁用也没用了, 重新弄了一次就OK了.
